holoviews/plotting/bokeh/callbacks.py

```diff
@@ -210,32 +206,23 @@ class CustomJSCallback(MessageCallback):
         }}
 
         // Initialize Comm
-        if ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null)) {{
-          var comm_manager = Jupyter.notebook.kernel.comm_manager;
-          var comm = HoloViewsWidget.comms["{comm_id}"];
-          if (comm == null) {{
-            comm = comm_manager.new_comm("{comm_id}", {{}}, {{}}, {{}});
-            comm.on_msg(on_msg);
-            comm_manager["{comm_id}"] = comm;
-            HoloViewsWidget.comms["{comm_id}"] = comm;
-          }}
-        }} else {{
+        comm = HoloViews.comm_manager.get_client_comm("{plot_id}", "{comm_id}", on_msg);
+        if (!comm) {{
           return
         }}
 
         // Initialize event queue and timeouts for Comm
-        var comm_state = HoloViewsWidget.comm_state["{comm_id}"];
+        var comm_state = HoloViews.comm_state["{comm_id}"];
         if (comm_state === undefined) {{
             comm_state = {{event_buffer: [], blocked: false, time: Date.now()}}
-            HoloViewsWidget.comm_state["{comm_id}"] = comm_state
+            HoloViews.comm_state["{comm_id}"] = comm_state
```

Comment:
Would rename to ``comm_status`` but I don't mind if that doesn't make it into this PR (though that would be good).

Reply:
Done.